mailbox/api/src/test/java/org/apache/james/mailbox/events/EventBusContract.java

```diff
@@ -75,6 +102,10 @@ default MailboxListener newListener() {
         return listener;
     }
 
+    default EventCallsVerifierMailboxListener newEventCallsListener() {
```

Comment:
This stuff is not explicit and seems highly bound to some testing logic... Maybe we can keep that in some local methods...